### PR TITLE
193 api endpoint test

### PR DESCRIPTION
**Pull request by sebastian-nehrdich**
This are the first steps towards the text-view. For the time being, just the main text on the left side is displayed. An api call for fetching the parallels of the current viewport is already implemented, but I disabled it because it relies on the 193-branch of the buddhanexus-backend. 
I moved some of the code from views/utility/views-common to views/utility/preprocessing to have a clearer separation. @TrebuhD  The code in views/utility/preprocessing is also relevant for the table-view. 
Also some code that is specific to the text-view was moved away from views-common. 
Furthermore I deleted the preprocessing-code from views-common that is depreciated by now.

### CODE REVIEW

File: src/views/textview/text-view-middle.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import { customElement, html, LitElement, property } from 'lit-element';
```
Comment (TrebuhD) [inline src/views/textview/text-view-middle.js]: I think there's another file with the same name? We should probably rename one of them.

**Comment by TrebuhD** [comment]
I've removed all the redundant files and reformatted them. Feel free to look at the commits. I left some comments, but I'll merge this now and we can solve that stuff later.

File: src/views/textview/text-view-old.js [hunk 2]
```diff
@@ -0,0 +1,336 @@
+import { html } from 'lit-element';
```
[the remark this answers is not in the document]
Comment (sebastian-nehrdich) [reply src/views/textview/text-view-old.js]: Yes, I think I forgot to add the #...#-pattern of the backup files of emacs to my .gitignore, sorry for this. Shouldn't end up on github.

File: src/views/textview/text-view-layout.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+const windowHeight = window.innerHeight - 430;
+const windowWidth = window.innerWidth - 30;
+
+
+const TextViewLayout = ({ fileName, textLeft }) => {
+  return html`
+    <table class="text-view-table">
+      ${TextViewLayoutHeader(fileName)}
+    </table>
+      <vaadin-split-layout class="top-level-split" style="max-width: ${windowWidth}px; height: ${windowHeight}px">
```
[the remark this answers is not in the document]
Comment (sebastian-nehrdich) [reply src/views/textview/text-view-layout.js]: OK, I can agree with this!

**Pull request merged**